class MyException(Exception):
    """Исключение при повторной записи в базу."""
